@xen-orchestra/lite/src/components/host/network/HostPifSidePanel.vue

```diff
@@ -0,0 +1,478 @@
+          </VtsCardRowKeyValue>
+          <!-- PIF STATUS -->
+          <VtsCardRowKeyValue>
+            <template #key>
+              {{ isBond ? $t('bond-status') : $t('pif-status') }}
+            </template>
+            <template #value>
+              <VtsConnectionStatus :status="pifStatus" />
+            </template>
+          </VtsCardRowKeyValue>
+          <!-- PHYSICAL INTERFACE STATUS -->
+          <VtsCardRowKeyValue>
+            <template #key>
+              {{ $t('physical-interface-status') }}
+            </template>
+            <template #value>
+              <VtsConnectionStatus :status="physicalInterfaceStatus" />
+            </template>
+          </VtsCardRowKeyValue>
+          <!-- VLAN -->
+          <VtsCardRowKeyValue>
+            <template #key>
+              {{ $t('vlan') }}
+            </template>
+            <template #value>
+              {{ getVlan }}
+            </template>
+            <template v-if="getVlan !== '-'" #addons>
+              <UiButtonIcon
+                v-tooltip="copied && $t('core.copied')"
+                :icon="faCopy"
+                size="medium"
+                accent="brand"
+                @click="copy(String(getVlan))"
+              />
+            </template>
+          </VtsCardRowKeyValue>
+          <!-- TAGS -->
+          <VtsCardRowKeyValue>
+            <template #key>
+              {{ $t('tags') }}
+            </template>
+            <template #value>
+              <div v-if="networkTags">{{ networkTags }}</div>
+              <div v-else class="tags">
+                <UiTag v-for="tag in networkTags" :key="tag" accent="info" variant="secondary">
+                  {{ tag }}
+                </UiTag>
+              </div>
+            </template>
+          </VtsCardRowKeyValue>
+        </div>
+      </UiCard>
+      <!-- NETWORK INFORMATION -->
+      <UiCard class="card">
+        <UiCardTitle>{{ $t('network-information') }}</UiCardTitle>
+        <div class="content">
+          <!-- IP ADDRESSES -->
+          <VtsCardRowKeyValue class="ip-addresses-container">
+            <template #key>
+              <div class="ip-addresses-title">{{ $t('ip-addresses') }}</div>
+            </template>
+            <template v-if="addressesIp.length > 0" #value>
+              <div v-for="(ip, index) in addressesIp" :key="ip" class="ip-addresses">
+                <span class="text-ellipsis">{{ ip }}</span>
+                <div>
+                  <UiButtonIcon
+                    v-if="ip !== '-'"
+                    v-tooltip="copied && $t('core.copied')"
+                    :icon="faCopy"
+                    size="medium"
+                    accent="brand"
+                    @click="copy(ip)"
+                  />
+                  <UiButtonIcon
+                    v-if="index === 0 && addressesIp.length > 1"
+                    :icon="faEllipsis"
+                    size="medium"
+                    accent="brand"
+                  />
```

Comment:
I think we can set the button as `disabled` with a tooltip since it does nothing right now:
```suggestion
                  <UiButtonIcon
                    v-if="index === 0 && addressesIp.length > 1"
                    :icon="faEllipsis"
                    size="medium"
                    accent="brand"
                    disabled
                  />
```